src/main/java/me/blackvein/quests/Quests.java

```diff
@@ -23,7 +23,8 @@
 import me.blackvein.quests.util.Lang;
 import me.blackvein.quests.util.MiscUtil;
 import net.aufdemrand.denizen.Denizen;
-import net.aufdemrand.denizen.scripts.ScriptRegistry;
+import net.aufdemrand.denizencore.DenizenCore;

```

Comment:
I don't know, I was just trying to copy what Blackvein did to fix functionality since it won't load any quests with scripts right now. See gists in #41.